apps/react-world/src/components/shared/Layout.tsx

```diff
@@ -1,6 +1,6 @@
 import { PropsWithChildren } from 'react';
-import { Navbar } from '../components/NavBar';
-import { Footer } from '../components/Footer';
+import { Navbar } from './NavBar';
+import { Footer } from './Footer';
 
 type LayoutProps = PropsWithChildren;
 
```

Comment:
제가 급하게 작성해서 이러한 코드 뭉치를 그대로 두었는데, 이러한 타입 alias 기법은 제3자로 하여금 그 의도를 흐리게 만들기 때문에 가급적 사용하지 않는게 좋긴 합니다🥲

```suggestion
type LayoutProps = PropsWithChildren<{}>;
```

차라리 이렇게 하거나...

```suggestion
type LayoutProps = {
  children?: ReactNode;
}
```

이렇게 하는 것을 추천드립니다 💪

Reply:
오 코멘트 감사합니다 저는 고르라면 후자쪽을 고를 것 같습니다 ㅎㅎ